src/Tizen.NET.MaterialComponents/Components/MNavigationView.cs

```diff
@@ -5,10 +5,12 @@
 
 namespace Tizen.NET.MaterialComponents
 {
-    public class MNavigationView : Box, IColorSchemeComponent
+    public class MNavigationView : Background, IColorSchemeComponent
```

Comment:
Why we need to inherit `Background` here?

Reply:
Maybe it was for background image. If so, is there any reason why  youI used `setPartContent` instead of `File` property?